### ISSUE TITLE
Improve: doesn't work with srcset

### ISSUE THREAD

**Issue by bars3s** [OverZealous/cdnizer#22]
Hi @OverZealous!

Found that `cdnizer` doesn't process `srcset` attributes. I think it will be very useful feature.

Thx!

**Comment by OverZealous** [comment]
Yeah, that definitely looks useful.  That being said, that's a lot of work to integrate: the current design uses a `RegExp`-based replacement strategy that expects a `(before)(url)(after)` format, while `srcset`s are a comma-separated list of values, each of which has to be processed individually.

I'll see if I can find some time to add it in the near future.
